test/pummel/test-net-connect-memleak.js

```diff
@@ -26,10 +26,10 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+console.log('Run this test with --expose-gc');
```

Comment:
Any reason for this? I think the "Flags" comment above is all we need.

Reply:
The comment is easy to miss. I actually missed it myself when I first ran this test. I thought console.log here behaves similarly to what you do with those unix command, where if you enter a wrong flag or miss a mandatory flag, it gives you a warning message on the console.

Reply:
@jwang-navarik The tests should be run with `tools/test.py`, running them with node is for debugging and that means you will need to look into the source code anyway, and the existing convention is `// Flags: --expose-gc` is the existing convention instead of `console.log()` so IMO we are good with that.